app/src/test/java/com/xpay/kotlinutils/XpayUtilsTest.kt

```diff
@@ -109,22 +142,22 @@ class XpayUtilsTest {
         }
     }
 
+    // pay returns error when payment method is not available in payment options
     @Test(expected = IllegalArgumentException::class)
     fun pay_paymentMethodNotAvailable_throwsError() {
         XpayUtils.apiKey = "3uBD5mrj.3HSCm46V7xJ5yfIkPb2gBOIUFH4Ks0Ss"
         XpayUtils.communityId = "zogDmQW"
         XpayUtils.variableAmountID = 18
         XpayUtils.PaymentOptionsTotalAmounts =
             PaymentOptionsTotalAmounts(cash = 50.0, kiosk = 52.85)
-//        XpayUtils.userInfo = User("Mahmoud Aziz", "mabdelaziz@xpay.app", "+201226476026")
         XpayUtils.payUsing = PaymentMethods.CARD
         XpayUtils.request = serviceRequest
         runBlocking {
             XpayUtils.pay()
         }
     }
 
-    // User information is not set
+    // pay returns error when user informations is missing
     @Test(expected = IllegalArgumentException::class)
     fun pay_userInfoNotSet_throwError() {
 
```

Comment:
you need to set available payment options otherwise it fails because it's empty not because the user information is missing